### PR TITLE
QAE-288 - E2E test case for off canvas menu item divider size and color option

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for off canvas menu item divider option

### Screenshots

https://share.getcloudapp.com/NQuxZXR0

### How has this been tested?

npm run test:e2e:interactive -- tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider-size-color.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../../../utils/customize';
+import { setBrowserViewport } from '../../../../../../utils/set-browser-viewport';
+describe( 'off canvas menu item divider settings in the customizer', () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider.test.js:4]: First letter should be capital

File: tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	it( 'off canvas menu item divider size and color should apply corectly for after header', async () => {
+		const Offcanvas = {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider.test.js:6]: constant name should be specific to test case

File: tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider.test.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../../../utils/customize';
+import { setBrowserViewport } from '../../../../../../utils/set-browser-viewport';
+describe( 'off canvas menu item divider settings in the customizer', () => {
+	it( 'off canvas menu item divider size and color should apply corectly for after header', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider.test.js:5]: @SimranChougule1  Color related setting and assertion is not present in this test case

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider.test.js]: added test case for divider size and color

File: tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider-size-color.test.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+			ppStatus = await publishPost();
+			await createNewPost( { postType: 'page', title: 'test-2' } );
+			ppStatus = await publishPost();
+		}
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await setBrowserViewport( 'medium' );
+		await page.click( '.main-header-menu-toggle' );
+		await page.waitForSelector( '.ast-hfb-header .ast-builder-menu-mobile .main-navigation .main-header-menu, .ast-hfb-header .ast-builder-menu-mobile .main-navigation .main-header-menu, .ast-hfb-header .ast-mobile-header-content .ast-builder-menu-mobile .main-navigation .main-header-menu' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider-size-color.test.js:25]: Please check and removed duplicate selectors:- https://share.bsf.io/wbuzPANk

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider-size-color.test.js]: Changed duplicate selector

File: tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider-size-color.test.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await setBrowserViewport( 'medium' );
+		await page.click( '.main-header-menu-toggle' );
+		await page.waitForSelector( '.ast-hfb-header .ast-builder-menu-mobile .main-navigation .main-header-menu' );
+		await expect( {
+			selector: '.ast-hfb-header .ast-builder-menu-mobile .main-navigation .main-header-menu',
+			property: 'border-top-width',
+		} ).cssValueToBe( `${ offCanvas[ 'header-mobile-menu-submenu-item-b-size' ] }` + 'px' );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider-size-color.test.js:29]: @SimranChougule1 test case is failing at local, please check once

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/item-divider-size-color.test.js:29]: @bsfsnehal Resolved PR comment

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**